tests/init_test.go

```diff
@@ -136,10 +136,6 @@ func (tm *testMatcher) fails(pattern string, reason string) {
 	tm.failpat = append(tm.failpat, testFailure{regexp.MustCompile(pattern), reason})
 }
 
-func (tm *testMatcher) whitelist(pattern string) {
```

Comment:
Hm, from time to time, I have to dig into the tests to find out why a particular one doesn't work, whenever the tests are updated. At those points in time, I heavily rely on this method -- it's  PITA to rerun every consensus test if I only want one. After it's all fixed, I remove that code again. 
So can we please leave it in, _and_ make the linter happy?

Reply:
I guess I can create a dummy test...